auctions/README.md

```diff
@@ -0,0 +1,115 @@
+# Auctions
+
+## Auction object
+
+```
+{
+  'id': <auction-id>,
+  'seller': <userId>,
+  'item_type': <example: neelam>,
+  'initial_price': <example: 100 dinero>,
+  'bidders_and_bids': [{
+    bidder: <userId>,
+    last_bid: <number>,
+    timestamp: <unix-timestamp>
+  }],
```

Comment:
I think we should keep the `highest_bid` attribute, as that is always displayed.
Otherwise, the front-end will always have to rerun the `max` function.

Reply:
Added it 👍🏻